sklearn/tree/tests/test_tree.py

```diff
@@ -135,10 +135,11 @@
 X_multilabel, y_multilabel = datasets.make_multilabel_classification(
     random_state=0, n_samples=30, n_features=10)
 
+# NB: these are sparse in nature, but X is still ndarray (and X_sparse is CSC)
```

Comment:
Comment is a bit confusing because it is not clear here what X_sparse is referring to. I thought it was actually referring to X_sparse_mix (but I gather that it refers to the 'X_sparse' key 20 lines below. Maybe the best thing to do is to remove "(and X_sparse is CSC)"?